R/randomForest_tidiers.R

```diff
@@ -640,6 +640,267 @@ augment.randomForest.unsupervised <- function(x, data, ...) {
 #' @export
 augment.randomForest <- augment.randomForest.formula
 
+#' augment for randomForest(ranger) model
+#' @export
+augment.ranger <- function(x, data = NULL, newdata = NULL, ...) {
+  # Extract data from model
+  # This is from https://github.com/mdlincoln/broom/blob/e3cdf5f3363ab9514e5b61a56c6277cb0d9899fd/R/rf_tidiers.R
+  if (is.null(data)) {
```

Comment:
In else part of these conditions, shouldn't we call stop()?

Reply:
Look OK after all, since it is the case for when data is not passed as argument.